File: DecimaI Comparator/DecimaIComparator.py
import math
def areEqualByThreeDecimalPlaces(a,b):
    a,a1 = math.modf(a*1000)
    b,b1 = math.modf(b*1000)
    if(a1 == b1):
        return True
    return False

print(areEqualByThreeDecimalPlaces(2.123,2.12344))

File: DecimaI Comparator/DecimaIComparator.js
function areEqualByThreeDecimalPlaces(a,b){
    a =a*1000;
    b= b*1000;
    if(Math.floor(a) == Math.floor(b))
        console.log(true);
    else
        console.log(false);
}

areEqualByThreeDecimalPlaces(1.2349233, 1.23232);
